docs/tutorial.rst

```diff
@@ -171,7 +171,8 @@ So we add the search results page definition to pages.py:
 
 .. literalinclude:: code/round_2/pages.py
     :language: python
-    :lines: 6-18
+    :lines: 1-18
+    :emphasize-lines: 2, 6-18
```

Comment:
This isn't showing up (click the "view" button at the top to see it rendered, https://github.com/billagee/bok-choy/blob/doc-edits/docs/tutorial.rst).

Also the code example at L142 (under " Add a method for filling in the search term to the page object definition like this:") isn't showing up